### ISSUE TITLE
Sync - recognise existing PRs and treat as success

### ISSUE THREAD

**Issue by ewels** [nf-core/tools#710]
See https://github.com/nf-core/tools/issues/709#issuecomment-666683931

Need to catch the following error:

```json
ERROR     Sync failed for nf-core/ampliseq:                          sync.py:385
GitHub API returned code 422:
{
    "message": "Validation Failed",
    "errors": [
        {
            "resource": "PullRequest",
            "code": "custom",
            "message": "A pull request already exists for nf-core:TEMPLATE."
        }
    ],
    "documentation_url": "https://developer.github.com/v3/pulls/#create-a-pull-request"
}
```

and treat it as a success - in this scenario the PR to update the template already exists, and our new commit is appended to it.

**Comment by ewels** [comment]
Ooh, even better - bioconda bot recognises this and then _renames_ the PR title:

For example: https://github.com/bioconda/bioconda-recipes/pull/23469

**Comment by ewels** [comment]
Closed in #716
